modules/andes-core/broker/src/main/java/org/wso2/andes/mqtt/connectors/PersistenceStoreConnector.java

```diff
@@ -94,6 +94,7 @@ public void addMessage(MQTTMessageContext messageContext) throws MQTTException {
                 publisherTopicCorrelate.put(messageContext.getPublisherID(), publisher);
                 //Finally will register the publisher channel for flow controlling
                 AndesChannel publisherChannel = Andes.getInstance().createChannel(publisher);
+                publisherChannel.setClientSocket(messageContext.getChannel().remoteAddress().toString().substring(1));
```

Comment:
can you add a comment what you doing here. since it looks like a magic number in 'substring(1)'